### PR TITLE
[PWN-7098] Create Swap Main screen view

### PR DESCRIPTION
**Pull request by lisemyon**
## Link jira to issue
https://p2pvalidator.atlassian.net/browse/PWN-7098

## Description of the changes
- Swap View
- Logic of auto-selection initial swap tokens
- Loading state
- Cache jupiter tokens for 1 day

### CODE REVIEW

**Review by bigearsenal — COMMENTED** [review]
I could not build the application

File: p2p_wallet/Scenes/Main/Swap/Providers/SwapWalletsRepository.swift
Comment (bigearsenal) [inline p2p_wallet/Scenes/Main/Swap/Providers/SwapWalletsRepository.swift:41]: `@Published` is a short-hand property wrappers for `CurrentValueSubject<ResultType, Never>`. I don't really get it why `stateSubject` and `dataSubject` is both `@Published` of type `CurrentValueSubject` because `@Published` has no effect in this context, the Subject itself never changes, only value changes.
May be you mean `@Published private var state = SwapWalletsState.loading` ?
Or `private var stateSubject = CurrentValueSubject<SwapWalletsState, Never>(.loading)`
They are equivalent

**Review by bigearsenal — COMMENTED** [review]
SwapRepository should be better named JupiterTokensRepository

**Review by bigearsenal — CHANGES_REQUESTED** [review]
Could not build the application

File: p2p_wallet/Scenes/Main/Swap/Providers/SwapWalletsRepository.swift
Comment (lisemyon) [reply p2p_wallet/Scenes/Main/Swap/Providers/SwapWalletsRepository.swift:41]: I looked through your repository solution and will integrate it as soon as it will be merged into develop

**Review by bigearsenal — APPROVED** [review]

File: p2p_wallet/Scenes/Main/Swap/Providers/SwapWalletsRepository.swift
Comment (lisemyon) [reply p2p_wallet/Scenes/Main/Swap/Providers/SwapWalletsRepository.swift]: I updated PR without your changes yet. Pls look and I will merge it after

**Pull request merged**
